apps/fly-my-rockets/src/app/rockets/rocket.model.ts

```diff
@@ -20,3 +21,9 @@ export interface Motor {
   manufacturer?: string;
   impulse?: string;
 }
+
+interface FirebaseDate {
+  nanoseconds: number;
+  seconds: number;
+  toDate: () => Date;
```

Comment:
There's a few other functions available on this object. Will fill them in as needed.